Formula/udunits.rb

```diff
@@ -18,29 +17,23 @@ class Udunits < Formula
     sha256 x86_64_linux:  "e05daef7e7f7ad70952a77ac4096e5a72f201c04be9c78b3c684b9846ab4cb9d"
   end
 
-  depends_on "cmake" => :build
+  head do
+    url "https://github.com/Unidata/UDUNITS-2.git", branch: "master"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
 
   uses_from_macos "bison" => :build
+  uses_from_macos "flex" => :build
   uses_from_macos "texinfo" => :build
   uses_from_macos "expat"
-  uses_from_macos "flex"
-
-  on_linux do
-    patch :p1 do
-      url "https://github.com/Unidata/UDUNITS-2/commit/0bb56200221ad960bc2da11fc0b4a70ec3c5d7c9.patch?full_index=1"
-      sha256 "8b84fabe21d2da252e6bdd2dd514230d73579ca034d4d83e42f40527dc72fe0c"
-    end
-  end
 
   def install
-    mkdir "build" do
-      system "cmake", "..", *std_cmake_args
-      system "make", "install"
-      system "make", "clean"
-      system "cmake", "..", *std_cmake_args, "-DBUILD_SHARED_LIBS=OFF"
-      system "make"
-      lib.install "lib/libudunits2.a"
-    end
+    system "autoreconf", "--verbose", "--install", "--force" if build.head?
+    system "./configure", *std_configure_args, "--disable-silent-rules"
```

Comment:
Autotools should build both shared and static libraries by default, so shouldn't need separate builds.

Existing bottle built from CMake:
```
/usr/local/Cellar/udunits/2.2.27.6_1/lib
├── libudunits2.a
└── libudunits2.dylib
```

Source build of this update built from Autotools:
```
/usr/local/Cellar/udunits/2.2.28/lib
├── libudunits2.0.dylib
├── libudunits2.a
└── libudunits2.dylib -> libudunits2.0.dylib
```